### ISSUE TITLE
Allow <video> tags on app's info.xml

### ISSUE THREAD

**Issue by SamuAlfageme** [owncloud/market#202]
I was discussing 'bout this with @hurradieweltgehtunter this past hackweek. Would be cool if publishers are able to include and showcase videos on their apps description easily. 

Take [the video @haukman recorded for the release of the OAuth2 app](https://www.youtube.com/watch?v=lZEuVB72BIk) as example. e.g. 

![screenshot 2017-12-13 13 29 50](https://user-images.githubusercontent.com/2644445/33938995-bdcfcf8e-e009-11e7-94fb-9840e6a172db.png)

Providing a `.mp4`/`.webm` source/youtube link would do; ideally we could reuse HTML's 5 [`<video>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) implementation to some point.

**Comment by PVince81** [comment]
Or make it a "media" tag that accepts multiple media types, picture or video. Then the UI of whatever is showing it can provide a slideshow view to scroll between the medias. Like the Google Play Store.

**Comment by SamuAlfageme** [comment]
@PVince81 yup; include the video in the slideshow that gets generated with the `<screenshot>` tags already - e.g. https://github.com/owncloud/oauth2/blob/master/appinfo/info.xml#L29-L31 

![screenshot 2017-12-13 15 37 44](https://user-images.githubusercontent.com/2644445/33944154-970b84bc-e01b-11e7-977e-c48a5b61b4e1.png)

**Comment by DeepDiver1975** [comment]
well - playing videos in the browser is still a pain with all the different supported formats.
Why not use animated png? Supported on all browsers, no need for any additional new tag and videos can we converted to animated pngs.

objecstions?

**Comment by mmattel** [comment]
I like the idea of having animated png as implenetation and usability seems to have no issue.
But to be fair, what are the limitations of animated png compared to true videos and are they effecting us. Additionally, are there eg file size limitations publishers need to consider? Like webspace ect or anything else?
The rest is a matter of documentation...
